examples/factors/Cargo.lock

```diff
@@ -0,0 +1,1856 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I'm surprised to see this file here, I would think it would live next to the workspace Cargo.toml

Reply:
Yep, this file isn't used in this location; removed.